tests/features/policy.feature

```diff
@@ -18,15 +23,38 @@ Feature: Policy content creation
     And I fill in "Legacy link" with "http://example.com"
     And I fill in "Navigation title" with "Navi title"
     And I fill in "Short title" with "Shorter title"
+    And I fill in "Label" with "Label 1" in the "first" "Timeline" field element
```

Comment:
Shouldn't label be removed from widget?

Reply:
We hide it in oe_theme in order to facilitate the ECLv2 implementation.